DatadogCrashReporting/Sources/PLCrashReporterIntegration/PLCrashReporterIntegration.swift

```diff
@@ -44,11 +44,21 @@ internal extension PLCrashReporterConfig {
 
 internal final class PLCrashReporterIntegration: ThirdPartyCrashReporter {
     private let crashReporter: PLCrashReporter
+    private let backtraceReporter: PLCrashReporter
     private let builder = DDCrashReportBuilder()
 
     init() throws {
-        self.crashReporter = try PLCrashReporter(configuration: .ddConfiguration())
+        let configuration: PLCrashReporterConfig = try .ddConfiguration()
```

Comment:
Suggestion/ I think there are parts of the configuration that we may want to differentiate for backtrace reporter, for example the path to write file (to not conflict with regular crashes) or disabling the uncaught exception handler in this instance (that if enabled in second instance, may cause troubles I believe).